pkgs/tools/backup/duplicati/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, mono, sqlite, makeWrapper, unzip }:
+
+stdenv.mkDerivation rec {
```

Comment:
Could you try building it from source?

Reply:
Yes, I want to do this in a following update to the package. Since it is my first attempt to create a package I wanted to start simple and improve later on.

Reply:
Default Makefile is using nuget to fetch dependencies, and it seems that it is not possible for a Nix build to fetch things from the internet during the build. I don't know enough about C# builds to know how this could work, so someone with more knowledge on this subject could help (maybe I should create an issue ?)